ml4h/models/merge_blocks.py

```diff
@@ -119,13 +122,16 @@ def __init__(
             pairs: List[Tuple[TensorMap, TensorMap]],
             pair_loss: str = 'cosine',
             pair_loss_weight: float = 1.0,
+            batch_size: int = 4,
             **kwargs,
     ):
         self.pairs = pairs
         if pair_loss == 'cosine':
             self.loss_layer = CosineLossLayer(pair_loss_weight)
         elif pair_loss == 'euclid':
             self.loss_layer = L2LossLayer(pair_loss_weight)
+        elif pair_loss == 'contrastive':
+            self.loss_layer = ContrastiveLossLayer(pair_loss_weight, batch_size)
 
```

Comment:
maybe include `else: raise ValueError(f"{pair_loss} is not a valid loss type")` after this incase of mistyped loss

Reply:
Done!